src/panels/lovelace/cards/hui-entities-card.js

```diff
@@ -117,19 +119,9 @@ class HuiEntitiesCard extends EventsMixin(PolymerElement) {
     for (let i = 0; i < entities.length; i++) {
```

Comment:
change this to `forEach()` please

Reply:
this was the original code but as you wish